contracts/EasyAuction.sol

```diff
@@ -109,6 +109,7 @@ contract EasyAuction is Ownable {
         uint256 feeNumerator;
         uint256 minFundingThreshold;
         bool minFundingThresholdNotReached;
+        bool isAtomicClosureAllowed;
```

Comment:
A minor efficiency gain would be moving the two bools after `volumeClearingPriceOrder`, so that `uint96`, `bool`, and `bool` can be packed into the same storage slot.